docs/user/addons.rst

```diff
@@ -46,8 +46,323 @@ Individual configuration options for each addon are available in :guilabel:`Sett
+
+    **Notifications**
+
+    - ``--readthedocs-notification-background-color``
+    - ``--readthedocs-notification-color``
+    - ``--readthedocs-notification-font-family``
+    - ``--readthedocs-notification-font-size``
+    - ``--readthedocs-notification-link-color``
+    - ``--readthedocs-notification-title-background-color``
+    - ``--readthedocs-notification-title-color``
+    - ``--readthedocs-notification-toast-font-size``
+
+    **Search**
+
+    - ``--readthedocs-search-backdrop-color``
+    - ``--readthedocs-search-color``
+    - ``--readthedocs-search-content-background-color``
+    - ``--readthedocs-search-content-border-color``
+    - ``--readthedocs-search-filters-border-color``
+    - ``--readthedocs-search-font-family``
+    - ``--readthedocs-search-font-size``
+    - ``--readthedocs-search-footer-background-color``
+    - ``--readthedocs-search-footer-code-background-color``
+    - ``--readthedocs-search-footer-code-border-color``
+    - ``--readthedocs-search-input-background-color``
+    - ``--readthedocs-search-result-section-border-color``
+    - ``--readthedocs-search-result-section-color``
+    - ``--readthedocs-search-result-section-highlight-color``
+    - ``--readthedocs-search-result-section-subheading-color``
+
+You can find default values and full CSS in our `Addons source <https://github.com/readthedocs/addons/tree/main/src>`_.
+
+Custom event integration
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Read the Docs provides a custom event ``readthedocs-addons-data-ready`` that allows you to access the Addons data and integrate it into your theme or documentation.
+The event provides access to the version data, project information, and other Addons configuration.
+
+To use the custom event:
+
+1. Add the required meta tag to your HTML template:
+
+   .. code-block:: html
+
+      <meta name="readthedocs-addons-api-version" content="1" />
+
+2. Add a JavaScript event listener to handle the data:
+
+   .. code-block:: javascript
+
+      document.addEventListener(
+        "readthedocs-addons-data-ready",
+        function (event) {
+          // Access the addons data
+          const config = event.detail.data();
+
+          // Example: Create a version selector
+          const versions = config.versions.active.map(version => ({
+            slug: version.slug,
+            url: version.urls.documentation
+          }));
+
+          // Use the data to build your UI
+          console.log('Available versions:', versions);
+        }
+      );
+
+Event data reference
+^^^^^^^^^^^^^^^^^^^^
+
+The event.detail.data() object contains all the Addons configuration, including:
+
+* ``addons`` - Individual addon configurations
+* ``builds.current`` - Details about the current build
+* ``projects.current`` - Current project details
+* ``projects.translations`` - Available translations
+* ``versions.current`` - Details about the current version
+* ``versions.active`` - List of all active versions
```

Comment:
This is the list of "active, built and not hidden" versions. We can explain that in the paragraph.

Reply:
```suggestion
* ``versions.active`` - List of all active and not hidden versions
```